logger.js

```diff
@@ -19,11 +19,12 @@
 const assert = require('assert')
 const jsonHelper = require('./json-helper')
 
-const Logger = (conf, scopedProperties) => {
-  assert.equal(typeof(conf.service), 'string',
+const Logger = (service_name,
```

Comment:
Minor but node convention is camelCase so should be `serviceName` in my opinion

Reply:
Fixed.